README.md

```diff
@@ -577,6 +579,70 @@ $data = [
 
 > [1, 2, 3, 3, 4, 5]
 
+### IfElse
+
+Delegates to one expression or another depending on whether the specified condition strictly evaluates to true.
+
+If the condition does not return a boolean an InvalidReturnException will be thrown.
+
+#### Signature
+
+```php
+IfElse(callable $condition, Strategy|Mapping|array|mixed $if, Strategy|Mapping|array|mixed $else = null)
+```
+
+ 1. `$condition` &ndash; Condition.
+ 2. `$if` &ndash; Expression used when condition loosely evaluates to true.
```

Comment:
I do not think *loosely* is correct here. Since we already expressed the comparison is strict in the lead, we can probably just omit this word here and on the line below.